client/src/leap/soledad/client/api.py

```diff
@@ -788,10 +805,16 @@ def change_passphrase(self, new_passphrase):
 
     def raw_sqlcipher_query(self, *args, **kw):
         """
-        Run a raw sqlcipher query in the local database.
+        Run a raw sqlcipher query in the local database, and return the result.
```

Comment:
Just to be exact, this returns a deferred that fires with the result, is that correct?